src/test/java/com/epam/steps/admin/CreateStudentSteps.java

```diff
@@ -0,0 +1,108 @@
+    private final StudentsPopup studentsPopup = new StudentsPopup();
+    private final CreatePopup createPopup = new CreatePopup();
+
+    @Then("See all elements are present in student section")
+    public void seeAllElementsArePresentInStudentSection() {
+        assertThat(adminPage.checkUIofStudentsSection())
+                .withFailMessage("Elements are not present in student section")
+                .isTrue();
+    }
+
+    @Then("Check all fields are present in create popup - students section")
+    public void checkAllFieldsArePresentInCreatePopupStudentsSection() {
+        assertThat(studentsPopup.checkUIOfCreatePopupStudentsSection())
+                .withFailMessage("Elements are not present in create popup - student section")
+                .isTrue();
+    }
+
+    @And("Fill in all required fields in students section create popup")
+    public void fillInAllRequiredFieldsInStudentsSectionCreatePopup() {
+        createPopup.fillNameSurnameEmail();
+        createPopup.clickOnGeneratePasswordButton();
+        studentsPopup.fillBloodGenderBirthdayAddress();
+    }
+
+    @Then("Check all input fields are empty in students section create popup")
+    public void checkAllInputFieldsAreEmptyInStudentsSectionCreatePopup() {
+        assertThat(studentsPopup.checkAllFieldsAreEmptyInStudentsCreatePopup())
+                .withFailMessage("All input fields are not empty in create popup - student section")
+                .isTrue();
+    }
+
+    @When("Fill in existed name, surname, all other required fields besides email")
+    public void fillInExistedNameSurnameNonExistedMailAllOtherRequiredFields() {
+        createPopup.fillExistedName();
+        createPopup.fillExistedSurname();
+        studentsPopup.fillBloodGenderBirthdayAddress();
+    }
+
+    @When("Click on 'Birth Day' field opens calendar")
+    public void clickOnBirthDayFieldOpensCalendar() {
+        assertThat(studentsPopup.checkCalendarIsOpened())
+                .withFailMessage("Calendar is not opened in create popup - students section")
+                .isTrue();
+    }
+
+    @Then("User is able to select dates between the given interval provided by documentation")
+    public void userIsAbleToSelectDatesBetweenTheGivenIntervalProvidedByDocumentation() {
+        assertThat(studentsPopup.isYearFromInterval())
+                .withFailMessage("Selected year is not from the given interval provided by documentation")
+                .isTrue();
+    }
+
+    @And("The user choice is displayed in the {} field")
+    public void theUserChoiceIsDisplayedInTheLinkedClassField(String fieldName) {
+        assertThat(studentsPopup.checkValueOfSelectedField(fieldName))
+                .withFailMessage("Saved value of selected fields is not the same with what user sees on screen")
+                .isTrue();
+    }
+
+    @Then("Check student created by admin is not added in the DB")
+    public void checkAdminIsNotAddedInTheDB() {
```

Comment:
Please, change the method name to "checkStudentIsNotAddedInTheDB()"

Reply:
done